spec/helpers/analytics_helper_spec.rb

```diff
@@ -0,0 +1,53 @@
+# -*- encoding : utf-8 -*-
+require File.expand_path(File.dirname(__FILE__) + '/../spec_helper')
+
+describe AnalyticsHelper do
+
+  include AnalyticsHelper
+
+  describe "#track_analytics_event" do
+    it "returns correctly formatted event javascript" do
+      expect(track_analytics_event(
+        AnalyticsEvent::Category::OUTBOUND,
+        AnalyticsEvent::Action::FACEBOOK_EXIT)).to eq(
+        "if (ga) { ga('send','event','Outbound Link','Facebook Exit') };"
+      )
+    end
+
+    context "when passed a label" do
+      it "includes the default value 1 if none is supplied" do
+        expect(track_analytics_event("test category", "test", :label => "test label")).to eq(
+          "if (ga) { ga('send','event','test category','test','test label',1) };"
+        )
+      end
+
+      it "uses the supplied value if there is one" do
+        expect(track_analytics_event(
+          "test category", "test", {:label => "test label", :value => 42}
+        )).to eq(
+          "if (ga) { ga('send','event','test category','test','test label',42) };"
+        )
+      end
+
+      it "doesn't wrap the label in quotes if passed label_is_script=true" do
+        expect(track_analytics_event(
+          "test category",
+          "test",
+          {:label => "window.location.href", :value => 42, :label_is_script => true}
+        )).to eq(
+          "if (ga) { ga('send','event','test category','test',window.location.href,42) };"
+        )
+      end
+
+      it "raises an ArgumentError if value is not nil or an Integer" do
```

Comment:
Just to clarify, I'm not sure what the intent of "is not nil" was here.

Given the expectation in this spec, I'd expect its description to read: `it "raises an ArgumentError if the :value option is not an Integer"`.

Should we be testing for some `nil` case in a new spec?